addons/product/models/product_product.py

```diff
@@ -30,11 +30,14 @@ class ProductProduct(models.Model):
         'Variant Price Extra', compute='_compute_product_price_extra',
         digits='Product Price',
         help="This is the sum of the extra price of all attributes")
-    # lst_price: catalog value + extra, context dependent (uom)
+    # lst_price: catalog value + extra, or custom value
     lst_price = fields.Float(
-        'Sales Price', compute='_compute_product_lst_price',
-        digits='Product Price', inverse='_set_product_lst_price',
-        help="The sale price is managed from the product template. Click on the 'Configure Variants' button to set the extra attribute prices.")
+        'Sales Price',
+        compute='_compute_product_lst_price',
+        digits='Product Price',
+        readonly=False,
+        store=True,
+        help="The sale price can be set manually or computed from the product template. Click on the 'Configure Variants' button to set the extra attribute prices.")
```

Comment:
not worth modifying since we'll try to merge it today for the freeze, but you could have reordered the field attributes according to our [team guidelines](https://www.odoo.com/odoo/knowledge/8790).